actors/builtin/multisig/testing.go

```diff
@@ -0,0 +1,73 @@
+package multisig
+
+import (
+	"bytes"
+	"encoding/binary"
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/v2/actors/builtin"
+	"github.com/filecoin-project/specs-actors/v2/actors/util/adt"
+)
+
+type StateSummary struct {
+	PendingTxns  uint64
+	NumApprovals uint64
+	Signers      int
+}
```

Comment:
Naming nit: name numbers/counts appropriately, leaving namespace for including the actual data later without an awkward name.

```suggestion
type StateSummary struct {
	PendingTxnCount  uint64
	NumApprovalsThreshold uint64
	SignerCount      int
}
```